extra/mariabackup/xtrabackup.cc

```diff
@@ -5408,6 +5408,69 @@ static void rename_force(const char *from, const char *to) {
 }
 
 
+/**
+ * Given an ibd file path, this returns the space id.
+ * @param file The ibd file
+ * @return space id for the ibd file, -1 if an error.
+ */
+static
+ulint get_space_id(const char *file) {
```

Comment:
InnoDB tablespace identifiers are 32-bit. The return type should be `uint32_t` accordingly.

`-1` is never returned by the function.